### PR TITLE
Fixes for new UI

### PR DESCRIPTION
**Pull request by Cyteon**
# Description

This should fix all issues with the new ui!

# Issue Fixes

Should fix:

- #1971

# Checklist:

- [V] I am pushing changes to the **develop** branch
- [V] I am using the recommended development environment
- [V] I have performed a self-review of my own code
- [V] I have commented my code, particularly in hard-to-understand areas
- [V] I have formatted and linted my code using python-black and pylint
- [V] I have cleaned up unnecessary files
- [V] My changes generate no new warnings
- [V] My changes follow the existing code-style
- [V] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

**Comment by Cyteon** [comment]
the memes support commit is not supposed to be there

**Comment by Cyteon** [comment]
reverted, ready to be merged

**Comment by LevaniVashadze** [comment]
Why is it not headless anymore?

File: video_creation/data/videos.json [hunk 1]
```diff
@@ -1 +0,0 @@
-[]
```
Comment (OpenSourceSimon) [inline video_creation/data/videos.json:1]: Can you revert this change? If I believe correctly, this will cause the bot to crash.

File: video_creation/screenshot_downloader.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
         print_substep("Launching Headless Browser...")
 
         browser = p.chromium.launch(
-            headless=True
+            headless=False
```
Comment (OpenSourceSimon) [inline video_creation/screenshot_downloader.py:76]:
```suggestion
            headless=True
```
Why would we set headless to False? Probably forgot to turn it back off when debugging :)

File: video_creation/screenshot_downloader.py [hunk 3]
```diff
@@ -218,7 +218,7 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
                 if page.locator('[data-testid="content-gate"]').is_visible():
                     page.locator('[data-testid="content-gate"] button').click()
 
-                page.goto(f'https://reddit.com{comment["comment_url"]}', timeout=0)
+                page.goto(f"https://new.reddit.com/{comment['comment_url']}")
```
Comment (OpenSourceSimon) [inline video_creation/screenshot_downloader.py:221]: Well found!

**Review by OpenSourceSimon — CHANGES_REQUESTED** [review]
Haven't tested it, but looks great! Visited the website new.reddit.com, looks exactly like the old lay-out!

File: video_creation/screenshot_downloader.py (hunk 2 above)
Comment (Cyteon) [reply video_creation/screenshot_downloader.py:76]: I was not sure if it was working headless, will check and If it works I'll change it back to True

Comment (Cyteon) [reply video_creation/screenshot_downloader.py:76]: Wont work in headless

**Comment by dsaderholm** [comment]
Hey, it seems like new.reddit.com with // is redirecting to old Reddit now. Am I the only one with this issue?

File: video_creation/screenshot_downloader.py (hunk 2 above)
Comment (OpenSourceSimon) [reply video_creation/screenshot_downloader.py:76]: > Wont work in headless

Why not? Are you sure? Reddit shouldn't be able to see that so I think it won't make a difference

**Comment by Faycal-S** [comment]
Behavior is not consistent for me unfortunately. It works sometimes but doesn't work sometimes :/
I noticed in headless=False mode that when it doesn't work, it's failing to login to reddit.. 
Any ideas?

**Comment by dsaderholm** [comment]
> Behavior is not consistent for me unfortunately. It works sometimes but doesn't work sometimes :/ I noticed in headless=False mode that when it doesn't work, it's failing to login to reddit.. Any ideas?

I am having the same issue. Sometimes it works, sometimes it doesn't.

**Comment by Cyteon** [comment]
Known issue

File: video_creation/screenshot_downloader.py (hunk 2 above)
Comment (Cyteon) [reply video_creation/screenshot_downloader.py:76]: Reddit knows somehow
